opentasks/src/main/java/org/dmfs/tasks/ViewTaskFragment.java

```diff
@@ -175,23 +176,23 @@ public void run()
          * @param data
          *         The task data that belongs to the {@link Uri}. This is purely an optimization and may be <code>null</code>.
          */
-        public void onEditTask(Uri taskUri, ContentSet data);
+        void onEditTask(Uri taskUri, ContentSet data);
 
         /**
          * This is called to inform the Activity that a task has been deleted.
          *
          * @param taskUri
          *         The {@link Uri} of the deleted task. Note that the Uri is likely to be invalid at the time of calling this method.
          */
-        public void onDelete(Uri taskUri);
+        void onDelete(Uri taskUri);
 
         /**
          * Notifies the listener about the list color of the current task.
          *
          * @param color
          *         The color.
          */
-        public void updateColor(int color);
+        void updateColor(@ColorInt int color);
```

Comment:
How about changing this to take a `Color` instead?

Reply:
I've changed it to use `Color`, also created https://github.com/dmfs/opentasks/issues/522 to use `Toned` as next step.
Ready for check.